apps/src/templates/SchoolAutocompleteDropdown.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
+
+export default class SchoolAutocompleteDropdown extends Component {
+
+  state = {};
+
+  render() {
+    const getOptions = (q) => {
+      if (!q.length) {
+        return Promise.resolve({ options: [{ value: -1, label: "School not found"}] });
+      }
+      return fetch('/dashboardapi/v1/schoolsearch/' + encodeURIComponent(q) + '/40')
+        .then((response) => response.json())
```

Comment:
Style nit: Omit parens here around `response` here.

> If your function takes a single argument and doesn’t use braces, omit the parentheses. Otherwise, always include parentheses around arguments for clarity and consistency.
> [airbnb JavaScript styleguide 8.4](https://github.com/airbnb/javascript#arrows--one-arg-parens)